content/codepipeline/forksample.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Fork Sample Repository"
+date: 2018-10-087T08:30:11-07:00
+weight: 12
+draft: true
+---
+
+We are now going to [fork](https://help.github.com/articles/fork-a-repo/) the sample Kubernetes service
+so that we will be able modify the repository and trigger builds.
+
+Login to GitHub and fork the sample service to your own account:
+
+[https://github.com/rnzsgh/eks-workshop-sample-api-service-go](https://github.com/rnzsgh/eks-workshop-sample-api-service-go)
```

Comment:
For continuity, I'd prefer we use one of the already existing workshop apps... perhaps the nodejs api? Changing it's output would be easy to see reflected through the main frontend.